crates/router/src/types/api/admin.rs

```diff
@@ -187,24 +183,22 @@ impl ForeignTryFrom<domain::BusinessProfile> for BusinessProfileResponse {
             })
             .transpose()?;
 
+        let order_fulfillment_time = item
+            .order_fulfillment_time
+            .map(|time| api_models::admin::OrderFulfillmentTime::new(time))
+            .transpose()
+            .change_context(errors::ParsingError::IntegerOverflow)?;
```

Comment:
It's not exactly integer overflow, right? It's that the `order_fulfillment_time` is not within the range 0 to 1800?